src/_slugline_fonts.scss

```diff
@@ -0,0 +1,9 @@
+@font-face {
+  font-family: "Body Font";
+  src: url("https://fonts.googleapis.com/css2?family=Merriweather&display=swap");
+}
+
+@font-face {
+  font-family: "Headings Font";
+  src: url('https://fonts.googleapis.com/css2?family=Overpass&display=swap');
```

Comment:
nit: replace single quotes with double quotes

Reply:
Missed that one, thanks.